sdk/ai/azure-ai-projects/samples/agents/user_functions.py

```diff
@@ -58,9 +58,114 @@ def send_email(recipient: str, subject: str, body: str) -> str:
     return message_json
 
 
+def calculate_sum(a: int, b: int) -> str:
```

Comment:
What will happen if you ask the sum of 1.1 + 2.2?

Reply:
Executing tool call: {'id': 'call_JluojhLbYo370qNz9HRWurod', 'type': 'function', 'function': {'name': 'calculate_sum', 'arguments': '{"a":1,"b":2}'}}
Tool outputs: [{'tool_call_id': 'call_JluojhLbYo370qNz9HRWurod', 'output': '{"result": 3}'}]

Reply:
I think the AI does try to do the best instead of error.